ui/app/controllers/jobs/index.js

```diff
@@ -240,7 +240,7 @@ export default class JobsIndexController extends Controller {
           console.log('error fetching job ids', e);
           this.notifyFetchError(e);
         }
-        if (this.jobs.length) {
+        if (this.jobs?.length) {
```

Comment:
Fix that I should've included in https://github.com/hashicorp/nomad/pull/23427 — errors from outside of Nomad (which I was experiencing when trying to switch regions with an invalid token) didn't yield jobs so I'd get a "cannot get .length of undefined" error